README.md

```diff
@@ -1,15 +1,60 @@
-# SSWG Logging API
 
-* Proposal: SSWG-xxxx
+# Proposal Review: SSWG-0001 (Server Logging API)
+
+After the great [discussion thread](https://forums.swift.org/t/server-logging-api/18834), we are proposing this as a final revision of this proposal and enter the proposal review phase which will run until the 20th January 2019.
+
+We have integrated most of the feedback from the discussion thread so even if you have read the previous version, you will find some changes that you hopefully agree with. To highlight a few of the major changes:
+
+- Logging meta-data is now structures and now supports nested dictionaries and list values, in addition to strings of course.
+- only locally relevant meta-data can now be passed to the individual log methods.
+- ship a multiplex logging solution with the API package
+
+The feedback model will be very similar to the one known from Swift Evolution. The community is asked to provide feedback in the way outlined below and after the review period finishes, the SSWG Steering Committee will -- based on the community feedback -- decide whether to promote the proposal to the [Sandbox](https://github.com/swift-server/sswg/blob/master/process/incubation.md#process-diagram) maturity level or not.
+
+### What goes into a review of a proposal?
+
+The goal of the review process is to improve the proposal under review through constructive criticism and, eventually, determine the evolution of the server-side Swift ecosystem.
+
+When reviewing a proposal, here are some questions to consider:
+
+- What is your evaluation of the proposal?
+
+- Is the problem being addressed significant enough to warrant a change to Swift?
+
+- Does this proposal fit well with the feel and direction of Swift?
+
+- If you have used other languages or libraries with a similar feature, how do you feel that this proposal compares to those?
+
+- How much effort did you put into your review? A glance, a quick reading, or an in-depth study?
+
+Thank you for contributing to the Swift Server Work Group!
+
+### What happens if the proposal gets accepted?
+
+If this proposal gets accepted, the official repository will be created and the code (minus examples, the proposal text, etc) will be submitted. The repository will then become usable as a SwiftPM package and a version (likely `0.1.0`) will be tagged. The development (in form of pull requests) will continue as a regular open-source project.
+
+---
+
+# Server Logging API
+
+* Proposal: SSWG-0001
 * Authors: [Johannes Weiss](https://github.com/weissi) & [Tomer Doron](https://github.com/tomerd)
-* Status: [Implemented](https://github.com/weissi/swift-server-logging-api-proposal)
+* Preferred initial maturity level: [Sandbox](https://github.com/swift-server/sswg/blob/master/process/incubation.md#process-diagram)
+* Name: `swift-server-logging-api`
```

Comment:
wonder about "swift-server", we received feedback that this may be useful for non-server use cases as well

Reply:
@tomerd I don't mind much, but it should match the repo name under github.com/apple that we'll create